packages/core/src/util/label.ts

```diff
@@ -54,3 +54,12 @@ export const createLabelDescriptionFrom = (withLabel: ControlElement): LabelDesc
     };
   }
 };
+
+export const translateLabel = (translations, label: LabelDescription): LabelDescription => {
+  if (translations && _.startsWith(label.text, '%')) {
+    const labelKey = label.text.substr(1, label.text.length);
+    label.text = translations[labelKey] ? translations[labelKey] : label.text;
```

Comment:
Should we bot at least remove the % and apply the default transformation?

Reply:
Jonas argued that we should stick with this behaviour in order to make it obvious for devs that a translation is missing. Personally, I'm fine with both approaches.